dbt_platform_helper/utils/platform_config_schema.py

```diff
@@ -0,0 +1,603 @@
+import ipaddress
+import re
+
+from schema import Optional
+from schema import Or
+from schema import Regex
+from schema import Schema
+from schema import SchemaError
+
+
+def _string_matching_regex(regex_pattern: str):
```

Comment:
from the error message, this function is a copilot specific validation so should probably be named as such

Reply:
It's actually only used to validate the branch name in the codebase pipeline, the bit of the error message was misleading. I have removed it.